deploy/olm-catalog/manifests/submariner-addon.clusterserviceversion.yaml

```diff
@@ -1,326 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Why was this file and the other two deleted?

Reply:
I didn't delete anything explicitly. I just bumped the version and ran `go mod tidy`. `go mod vendor`. All the other changes are automatic.

Reply:
But the commit is intended to bump the **submariner-operator** version so how/why did that result in changes to the add-on manifest/deployment? Specifically the `controller-gen.kubebuilder.io/version` was bumped in the yaml files - did you specifically change that? If not then what specific change in the _go.mod_  did it and is it desired?